<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MAFtranslator</class>
 <widget class="QWidget" name="MAFtranslator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MAF preprocessor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">

QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}

</string>
     </property>
     <property name="title">
      <string>input/output</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>input MAF file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineMAF"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseMAF">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>output data matrix</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineMatrix"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseMatrix">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">

QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}

</string>
     </property>
     <property name="title">
      <string>mutation types' selection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Custom set of mutation types to be used:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="mutationText">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="standardCheckBox">
          <property name="text">
           <string>use standard set</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="allCheckBox">
          <property name="text">
           <string>use all mutations</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">

QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}

</string>
     </property>
     <property name="title">
      <string>genes' selection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Custom set of genes to be used:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textCustomGenes">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CheckAllGenes">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Use all genes</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="translateButton">
     <property name="text">
      <string>Coumpute data matrix</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>browseMAF</sender>
   <signal>clicked()</signal>
   <receiver>MAFtranslator</receiver>
   <slot>getInput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>723</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseMatrix</sender>
   <signal>clicked()</signal>
   <receiver>MAFtranslator</receiver>
   <slot>setOutput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>723</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>translateButton</sender>
   <signal>clicked()</signal>
   <receiver>MAFtranslator</receiver>
   <slot>translate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>616</y>
    </hint>
    <hint type="destinationlabel">
     <x>723</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>allCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MAFtranslator</receiver>
   <slot>setAll(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>723</x>
     <y>481</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>standardCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MAFtranslator</receiver>
   <slot>setStandard(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckAllGenes</sender>
   <signal>clicked()</signal>
   <receiver>MAFtranslator</receiver>
   <slot>setAllGenes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getInput()</slot>
  <slot>setOutput()</slot>
  <slot>translate()</slot>
  <slot>setAll(bool)</slot>
  <slot>setStandard(bool)</slot>
  <slot>setAllGenes()</slot>
  <slot>setNumberOfGenes()</slot>
  <slot>setCustomGenes()</slot>
 </slots>
</ui>
